static/js/components/SiteContentListing.tsx

```diff
@@ -31,8 +34,18 @@ export default function SiteContentListing(): JSX.Element | null {
   }
 
   return isRepeatableCollectionItem(configItem) ? (
-    <RepeatableContentListing configItem={addDefaultFields(configItem)} />
+    <>
+      <RepeatableContentListing configItem={addDefaultFields(configItem)} />
+      <DocumentTitle
+        title={formatTitle(website.title, capitalize(pluralize(contenttype)))}
```

Comment:
Works great in most cases, but for the "Video Gallery" section of [one imported OCW site](https://ocw-studio-rc.odl.mit.edu/sites/10-805j-technology-law-and-the-working-environment-spring-2006/type/video_gallery/), the title includes "Video_galleries".  Maybe this could be modified to split on underscores and capitalize each word ("Video Galleries")?

Reply:
good catch! I'll add that